packages/destination-actions/src/destinations/stackadapt-audiences/index.ts

```diff
@@ -62,6 +62,24 @@ const destination: DestinationDefinition<Settings> = {
       }
     }
   },
+  onDelete: async (request, { payload }) => {
+    const userId = payload.userId ?? payload.anonymousId
+    // TODO: Add setting for advertiser ID and replace hardcoded value with it
+    const query = `mutation {
+      deleteProfiles(
+        subAdvertiserId: 1,
+        externalProvider: "Segment",
```

Comment:
```suggestion
        externalProvider: "segmentio",
```

Reply:
There are multiple places in the existing code this change will need to be made, I made them in https://github.com/StackAdapt/action-destinations/pull/5/commits/5fcb266ffd6f65f3b814516c1b72c11f1438843a